apps/web/src/pages/DNDDigitalCareersPage/jobFairs.ts

```diff
@@ -0,0 +1,186 @@
+import { defineMessages, IntlShape } from "react-intl";
+
+import {
+  DATE_FORMAT_LONG_STRING,
+  formatDate,
+} from "@gc-digital-talent/date-helpers";
+
+interface JobFair {
+  title: string;
+  lang?: string;
+  href: {
+    en: string;
+    fr: string;
+  };
+  date: string;
+  location: string;
+}
+
+interface FormattedDateArgs {
+  y: number;
+  m: number;
+  d: number;
+  intl: IntlShape;
+}
+
+const formattedDate = ({ y, m, d, intl }: FormattedDateArgs) =>
+  formatDate({
+    date: new Date(y, m, d),
+    intl,
+    formatString: DATE_FORMAT_LONG_STRING,
+  });
+
+const locations = defineMessages({
+  montreal: {
+    defaultMessage: "Montreal, Quebec",
+    id: "3AZTaf",
+    description: "Location name of Montreal, Quebec",
+  },
+  ottawa: {
+    defaultMessage: "Ottawa, Ontario",
+    id: "FxOKfM",
+    description: "Location name of Ottawa, Ontario",
+  },
+});
+
+const getJobFairs = (intl: IntlShape): JobFair[] => [
+  {
+    title: intl.formatMessage({
+      defaultMessage: "TechFair – McGill University",
+      id: "s0bXMy",
+      description: "Heading for TechFair job fair",
+    }),
+    href: {
+      en: "https://www.mcgill.ca/careers4engineers/techfair",
+      fr: "https://www.mcgill.ca/careers4engineers/fr/techfair",
+    },
+    date: intl.formatMessage({
+      defaultMessage: "September 25 and 26, 2025",
+      id: "9Qc9PO",
+      description: "Date for TechFair",
+    }),
+    location: intl.formatMessage(locations.montreal),
+  },
+  {
+    title: intl.formatMessage({
+      defaultMessage: "Career Expo University of Ottawa",
+      id: "q5nhMd",
+      description: "Heading for TechFair job fair",
+    }),
+    href: {
+      en: "https://www.uottawa.ca/en/events-all/career-expo",
+      fr: "https://www.uottawa.ca/fr/tous-evenements/expo-carrieres",
+    },
+    date: formattedDate({ y: 2025, m: 10, d: 2, intl }),
+    location: intl.formatMessage(locations.ottawa),
+  },
+  {
+    title: intl.formatMessage({
+      defaultMessage: "Fall Career & Networking Fair – Carleton University",
+      id: "zSvwW5",
+      description: "Heading for Carleton networking/carrer fair",
+    }),
+    href: {
+      en: "https://carleton.ca/career/events/career-fairs/",
+      fr: "https://carleton.ca/career/events/career-fairs/",
+    },
+    date: formattedDate({ y: 2025, m: 10, d: 15, intl }),
+    location: intl.formatMessage(locations.ottawa),
+  },
+  {
+    title: intl.formatMessage({
+      defaultMessage: "Foire de l’emploi - La Cité",
+      id: "gy65G/",
+      description: "Heading for Foire de l’emploi fair",
+    }),
+    lang: "fr",
+    href: {
+      en: "https://www.collegelacite.ca/foire-de-emploi-etudiants",
+      fr: "https://www.collegelacite.ca/foire-de-emploi-etudiants",
+    },
+    date: formattedDate({ y: 2025, m: 10, d: 22, intl }),
+    location: intl.formatMessage(locations.ottawa),
+  },
+  {
```

Comment:
Is the Figma out of date or? Because a bunch of the date values aren't matching

<img width="930" height="434" alt="image" src="https://github.com/user-attachments/assets/a5e094fb-c115-4bbb-80e3-991f8e9a8d88" />

Rendered

<img width="1228" height="349" alt="image" src="https://github.com/user-attachments/assets/7bb1f584-7f4e-4f6d-bbfb-75505ab050ad" />

Reply:
Oh no, I just forogot to remove one from the month for the `0` index :sweat_smile: Fixed in [fix dates](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14556/commits/61008d5e24d8f1d84cf5233b889e5a30d687d975)